lib/ApiRequestor.php

```diff
@@ -145,16 +145,26 @@ private static function _defaultHeaders($apiKey)
 
         $langVersion = phpversion();
         $uname = php_uname();
-        $curlVersion = curl_version();
+
+        $httplib = 'curl not installed';
```

Comment:
Hm, if `curl_version` doesn't exist we pretty much have no idea what they'll be using right? For that reason I was leaning toward just `unknown`.